src/Cake.DocFx.Tests/Build/DocFxBuildRunnerTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Cake.DocFx.Tests.Build
+﻿using Cake.Core.IO;
```

Comment:
@reicheltp I removed the unused using. Will cleanup the remaining usings directly on develop.